zkdoc/release-note

```diff
@@ -30,31 +30,42 @@ ZK 9.6.3
+  ZK-5170: borderlayout title css rules are missing under sapphire theme
+  ZK-5140: ZK loads vulnerable commons-io transitively
+  ZK-5220: a listbox without ROD causes a websocket connection to close
+  ZK-5029: sporadic text wrapping in listbox/grid/tree cells since chrome 90
+  ZK-5159: Firefox Windows 11 scrollbar doesn't have height, cause missing scrollbar
+  ZK-5115: Websocket endpoint registers threadpool, doesn't close, prevents tomcat graceful stop, threads remain after stop
+  ZK-5179: Custom attributes on template do not resolve after ZK-5002
   ZK-2259: Tabbox doesn't select second tab automatically when first tab is invisible
   ZK-5214: Locales.getLocale("de_DE_dgl") returns a different locale than zk 8
+  ZK-5253: title of borderlayout north/south shouldn't rotate 90 degrees
+  ZK-5240: calculation issue in tpad cause grid to render entire content at once
   ZK-5247: dynamically rerendered row breaking hflex min column
   ZK-5033: infinite loop when firing an event on a detached widget
   ZK-4974: grid-column with min-width
   ZK-5030: column header disappearing when resizing column
   ZK-5111: require.js causes Toast.show() failed
+  ZK-5223: the if element doesn't keep the position of its child component
+  ZK-5176: signature backgroundImage doesn't accept an absolute path
+  ZK-4982: Listbox (mold select with groups) fail when selecting from serverside
   ZK-5153: window with vflex content broken after title update
   ZK-5167: showBusy() doesn't mask the whole page
   ZK-5168: height calculation in groupbox with css scale incorrect after ZK-4670
   ZK-5177: signature saves a background image into a larger one
   ZK-5181: Optimize the use of DefinitionNotFoundException when creating event queues
   ZK-5148: datebox throws a js error with locale specified
+  ZK-5156: locale format doesn't keep the fractional part
+  ZK-5180: Bandbox closes its popup when clicking on day of a datebox
   ZK-5205: ClientInfoEvent should preserve the precision of devicePixelRatio
   ZK-5203: a highlighted window in an invisible parent doesn't get onSize event
+  ZK-5116: NullPointerException happened during evaluating an EL expression
+  ZK-5256: classloading issues with multiple war files
   ZK-5208: Review use of getChildNodeCount in AbstractTreeModel getChildNodes
   ZK-5215: cannot load locale-dependent resources in a relative path
-  ZK-5115: Websocket endpoint registers threadpool, doesn't close, prevents tomcat graceful stop, threads remain after stop
-  ZK-5179: Custom attributes on template do not resolve after ZK-5002
-  ZK-5220: a listbox without ROD causes a websocket connection to close
-  ZK-5256: classloading issues with multiple war files
   ZK-3543: Can't wire <apply> with its ID
-  ZK-5140: ZK loads vulnerable commons-io transitively
-  ZK-5223: the if element doesn't keep the position of its child component
-  ZK-5116: NullPointerException happened during evaluating an EL expression
```

Comment:
The lines above are removed starting from the first commit of this PR, so that there wouldn't be duplicate issues in release-note.